libraries/api-extractor/src/test/DocElementParser.test.ts

```diff
@@ -284,7 +284,25 @@ describe('DocElementParser tests', function (): void {
         +  ' contains spaces, encode them using %20; for display text, use a pipe delimiter ("|")']);
     });
 
-    it('Should parse @link with API defintion reference', (): void => {
+    it('Should reject @link with bad display text character', (): void => {
```

Comment:
I forgot to push it.  See below

---
In reply to: [122821868](https://github.com/Microsoft/web-build-tools/pull/240#discussion_r122821868) [](ancestors = 122821868)